src/tree/ManagedEnvironmentTreeItem.ts

```diff
@@ -3,166 +3,60 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { ContainerApp, ContainerAppsAPIClient, ManagedEnvironment } from "@azure/arm-appcontainers";
-import { LocationListStep, uiUtils, VerifyProvidersStep } from "@microsoft/vscode-azext-azureutils";
-import { AzExtParentTreeItem, AzExtTreeItem, AzureWizard, AzureWizardExecuteStep, AzureWizardPromptStep, DialogResponses, IActionContext, ICreateChildImplContext, parseError, TreeItemIconPath, UserCancelledError } from "@microsoft/vscode-azext-utils";
-import { ProgressLocation, window } from "vscode";
-import { ContainerAppCreateStep } from "../commands/createContainerApp/ContainerAppCreateStep";
-import { ContainerAppNameStep } from "../commands/createContainerApp/ContainerAppNameStep";
-import { EnableIngressStep } from "../commands/createContainerApp/EnableIngressStep";
-import { EnvironmentVariablesListStep } from "../commands/createContainerApp/EnvironmentVariablesListStep";
-import { IContainerAppContext } from "../commands/createContainerApp/IContainerAppContext";
-import { ContainerRegistryListStep } from "../commands/deployImage/ContainerRegistryListStep";
-import { azResourceContextValue, webProvider } from "../constants";
-import { ext } from "../extensionVariables";
-import { createContainerAppsAPIClient } from "../utils/azureClients";
-import { getResourceGroupFromId } from "../utils/azureUtils";
+import { ManagedEnvironment } from "@azure/arm-appcontainers";
+import { AzExtParentTreeItem, AzExtTreeItem, IActionContext, ICreateChildImplContext, TreeItemIconPath } from "@microsoft/vscode-azext-utils";
 import { localize } from "../utils/localize";
-import { nonNullProp } from "../utils/nonNull";
-import { settingUtils } from "../utils/settingUtils";
-import { treeUtils } from "../utils/treeUtils";
-import { ContainerAppTreeItem } from "./ContainerAppTreeItem";
 import { IAzureResourceTreeItem } from './IAzureResourceTreeItem';
+import { ResolvedContainerAppsResource } from "./ResolvedContainerAppsResource";
 
 export class ManagedEnvironmentTreeItem extends AzExtParentTreeItem implements IAzureResourceTreeItem {
-    public static contextValue: string = 'managedEnvironment';
-    public static contextValueRegExp: RegExp = new RegExp(ManagedEnvironmentTreeItem.contextValue);
-    public readonly contextValue: string = `${ManagedEnvironmentTreeItem.contextValue}|${azResourceContextValue}`;
+    public static contextValue: string;
```

Comment:
I think we should leave the initialization of these static ones here instead of the constructor, i.e.
```suggestion
    public static contextValue: string = ResolvedContainerAppsResource.contextValue;
```